themes/app/src/scss/_variables.scss

```diff
@@ -1,8 +1,6 @@
 /**
  * Application variables
  */
-
-$contentPadding: 20px 40px;
```

Comment:
why was this removed?

Reply:
Cause it's no longer used, since using Bootstrap.

Reply:
👍